src/step-web-index.js

```diff
@@ -2,5 +2,131 @@
  * step 3의 시작점이 되는 파일입니다.
  * 노드 환경에서 사용하는 readline 등을 불러올 경우 정상적으로 빌드할 수 없습니다.
  */
+import "./css/index.css";
+import "./js/index";
+import {
+  buyLottos,
+  calculateProfitRate,
+  getLottoRanks,
+} from "./js/domain/LottoService";
+import { LottoStats } from "./js/domain/LottoStats";
+import { LottoRank } from "./js/domain/enum/LottoRank";
+import { onModalShow } from "./js/index";
 
-console.log("Web Browser!");
+const $buyButton = document.getElementById("buyButton");
+const $moneyInput = document.getElementById("moneyInput");
+const $totalCountLabel = document.getElementById("totalCountLabel");
+const $lottosDiv = document.getElementById("lottosDiv");
+const $numbersToggle = document.getElementById("numbersToggle");
+const $winningNumberInputs = document.getElementsByClassName("winning-number");
+const $bonusNumberInput = document.getElementsByClassName("bonus-number")[0];
+const $resultButton = document.getElementById("resultButton");
+
+const $fifthRankCountSpan = document.getElementById("fifthRankCount");
+const $fourthRankCountSpan = document.getElementById("fourthRankCount");
+const $thirdRankCountSpan = document.getElementById("thirdRankCount");
+const $secondRankCountSpan = document.getElementById("secondRankCount");
+const $firstRankCountSpan = document.getElementById("firstRankCount");
+const $profitRateSpan = document.getElementById("profitRate");
+
+const $restartButton = document.getElementById("restartButton");
+
+let lottos = null;
+
+$buyButton.addEventListener("click", buyLottosEvent);
+
+$moneyInput.addEventListener("keydown", (event) => {
+  if (event.key === "Enter") {
+    event.preventDefault();
+    buyLottosEvent();
+  }
+});
```

Comment:
buyLottosEvent가 클릭 이벤트와 엔터키 입력 이벤트에 중복으로 등록되고 있네요. html form 요소를 사용하시면 엔터키 입력 시 자동으로 [submit 이벤트](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event)가 발생해 직접 엔터키 입력이나 클릭 이벤트에 리스너를 달지 않아도 제출 로직을 구현할 수 있어요 😄

```suggestion
$purchaseAmountform.addEventListener("submit", (event) => {
  event.preventDefault();
  buyLottosEvent();
});
```